LICENSE.md

```diff
@@ -0,0 +1,6 @@
+## CC-BY 4.0 License
+Copyright (c) 2016-2019 Uphold
```

Comment:
Update the year range to cover 2020 😄

Reply:
Or use `present`?

Reply:
I've added a note in our internal board to discuss this with the team so we can standardize copyright notices across all our repos. But sure, no opposition to do that here.